rust/tests/read_delta_test.rs

```diff
@@ -187,3 +189,33 @@ async fn read_delta_8_0_table_with_partitions() {
         ]
     );
 }
+
+#[tokio::test]
+async fn vacuum_delta_8_0_table() {
+    let table = deltalake::open_table("./tests/data/delta-0.8.0")
+        .await
+        .unwrap();
+
+    let retention_hours = 169;
+
+    assert_eq!(
+        table.vacuum_dry_run(retention_hours).unwrap(),
+        vec![format!["./tests/data/delta-0.8.0{}part-00001-911a94a2-43f6-4acb-8620-5e68c2654989-c000.snappy.parquet", MAIN_SEPARATOR]]
```

Comment:
Since there's already `/` in the path, I'm curious what the objective for the `format!` with `MAIN_SEPARATOR` is?

Reply:
It seems that the Windows CI test step doesn't like the `/` in the path 😆

Reply:
we could use `join_paths` to construct the path here right? https://github.com/delta-io/delta-rs/blob/main/rust/src/storage/mod.rs#L266